[   51s] /usr/lib64/gcc/x86_64-suse-linux/13/../../../../x86_64-suse-linux/bin/ld: /tmp/cc2L3fxe.ltrans21.ltrans.o: in function `Wt::WGLWidget::render(Wt::WFlags<Wt::RenderFlag>)':
[   51s] /home/abuild/rpmbuild/BUILD/wt-4.10.4/src/Wt/WServerGLWidget.C:120:(.text+0x8608): undefined reference to `XOpenDisplay'
[   51s] /usr/lib64/gcc/x86_64-suse-linux/13/../../../../x86_64-suse-linux/bin/ld: /tmp/cc2L3fxe.ltrans21.ltrans.o:/home/abuild/rpmbuild/BUILD/wt-4.10.4/src/Wt/WServerGLWidget.C:168:(.text+0x86eb): undefined reference to `XFree'
[   51s] /usr/lib64/gcc/x86_64-suse-linux/13/../../../../x86_64-suse-linux/bin/ld: /tmp/cc2L3fxe.ltrans21.ltrans.o:/home/abuild/rpmbuild/BUILD/wt-4.10.4/src/Wt/WServerGLWidget.C:187:(.text+0x872f): undefined reference to `XSync'
[   51s] /usr/lib64/gcc/x86_64-suse-linux/13/../../../../x86_64-suse-linux/bin/ld: /tmp/cc2L3fxe.ltrans21.ltrans.o:/home/abuild/rpmbuild/BUILD/wt-4.10.4/src/Wt/WServerGLWidget.C:201:(.text+0x8771): undefined reference to `XSync'
[   51s] /usr/lib64/gcc/x86_64-suse-linux/13/../../../../x86_64-suse-linux/bin/ld: /tmp/cc2L3fxe.ltrans21.ltrans.o:/home/abuild/rpmbuild/BUILD/wt-4.10.4/src/Wt/WServerGLWidget.C:213:(.text+0x8992): undefined reference to `XCloseDisplay'
[   51s] /usr/lib64/gcc/x86_64-suse-linux/13/../../../../x86_64-suse-linux/bin/ld: /tmp/cc2L3fxe.ltrans21.ltrans.o:/home/abuild/rpmbuild/BUILD/wt-4.10.4/src/Wt/WServerGLWidget.C:205:(.text+0x89b3): undefined reference to `XCloseDisplay'
[   51s] /usr/lib64/gcc/x86_64-suse-linux/13/../../../../x86_64-suse-linux/bin/ld: /tmp/cc2L3fxe.ltrans21.ltrans.o:/home/abuild/rpmbuild/BUILD/wt-4.10.4/src/Wt/WServerGLWidget.C:189:(.text+0x89bd): undefined reference to `XCloseDisplay'
[   51s] /usr/lib64/gcc/x86_64-suse-linux/13/../../../../x86_64-suse-linux/bin/ld: /tmp/cc2L3fxe.ltrans21.ltrans.o:/home/abuild/rpmbuild/BUILD/wt-4.10.4/src/Wt/WServerGLWidget.C:161:(.text+0x89ca): undefined reference to `XCloseDisplay'
[   51s] /usr/lib64/gcc/x86_64-suse-linux/13/../../../../x86_64-suse-linux/bin/ld: /tmp/cc2L3fxe.ltrans99.ltrans.o: in function `Wt::WServerGLWidget::~WServerGLWidget()':
[   51s] /home/abuild/rpmbuild/BUILD/wt-4.10.4/src/Wt/WServerGLWidget.C:225:(.text+0x45e7): undefined reference to `XCloseDisplay'
[   51s] collect2: error: ld returned 1 exit status

Index: wt-4.10.4/src/CMakeLists.txt
===================================================================
--- wt-4.10.4.orig/src/CMakeLists.txt
+++ wt-4.10.4/src/CMakeLists.txt
@@ -647,7 +647,7 @@ ELSE("${WT_WRASTERIMAGE_IMPLEMENTATION}"
 ENDIF("${WT_WRASTERIMAGE_IMPLEMENTATION}" STREQUAL "GraphicsMagick")
 
 if(WT_USE_OPENGL)
-  target_link_libraries(wt PRIVATE OpenGL::GL GLEW::GLEW)
+  target_link_libraries(wt PRIVATE OpenGL::GL GLEW::GLEW X11)
 else()
   MESSAGE("** Disabling server-side opengl rendering for WGLWidget: requires gl, glew and WRasterImage (GraphicsMagick or Direct2D).")
 endif()
